gr-azure-software-radio/python/integration_default_credentials.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
```

Comment:
New python file: Does this pass pylint?

Reply:
Steve agreed to fix this with his merge.